modules/new_profile/php/new_profile.class.inc

```diff
@@ -238,6 +239,9 @@ class New_Profile extends \NDB_Form
                 $errors['edcDate'] = 'Expected Date of Confinement is not a '
                     . 'valid date.';
             }
+        } else if (empty($values['dobDate'])) {
+            $errors['dobDate'] = "Date of Birth cannot be empty
+            if Expected Date isn't configured.";
```

Comment:
I don't think the "if" part should be in this error message. Users don't know anything about the study's LORIS configuration, so it's confusing and unnecessary.

Reply:
@driusan fixed in latest commit